test/link-fragments.md

```diff
@@ -86,6 +86,12 @@
 
 [Valid][escapedref]
 
+[Valid](#L7)
+
+[Valid](#L12-abc)
+
+### L12 ABC
```

Comment:
Nitpick, but the headings are in roughly the same order as the links, so this belongs near the end of the valid heading list.